src/block_cache/block_chunkcache.c

```diff
@@ -722,6 +722,8 @@ __chunkcache_read_into_chunk(
     /* Make sure the chunk is considered invalid when reading data into it. */
     WT_ASSERT(session, !new_chunk->valid);
 
+    __wt_capacity_throttle(session, new_chunk->chunk_size, WT_THROTTLE_CHUNKCACHE);
```

Comment:
Shall we restrict this to the FILE type only?

Reply:
I don't think there's any need - the in-memory version is only used for testing, so there's no harm in allowing throttling. (Plus it would mean the throttling test code needs to be different for those two configs.)